requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
We use python3.8 venv in [jenkins schema builder job](http://jenkins-new.analytics.edx.org/view/Warehouse/job/snowflake-schema-builder/)

[https://github.com/edx/jenkins-job-dsl/blob/master/dataeng/jobs/analytics/SnowflakeSchemaBuilder.groovy#L48](https://github.com/edx/jenkins-job-dsl/blob/master/dataeng/jobs/analytics/SnowflakeSchemaBuilder.groovy#L48)

I am not sure how we should cater update in python version.

Reply:
Should we pin python version as well ?

Reply:
Yes, I think we should pin python version